src/ssl.c

```diff
@@ -4565,6 +4543,26 @@ int ProcessBuffer(WOLFSSL_CTX* ctx, const unsigned char* buff,
         }
     }
 
+    if (ssl && resetSuites) {
+        int havePSK = 0;
+        int haveRSA = 0;
+
+        #ifndef NO_PSK
+        if (ssl->options.havePSK) {
+            havePSK = 1;
+        }
+        #endif
+        #ifndef NO_RSA
+            haveRSA = 1;
+        #endif
+
+        /* let's reset suites */
+        InitSuites(ssl->suites, ssl->version, haveRSA, havePSK,
```

Comment:
windows compiler is asking for a typecast here

(word16) haveRSA
(word16) havePSK

Doesn't like when we pass an Int (32-bit type) to word16 in InitSuites.